Terminal.Gui/ConsoleDrivers/CursesDriver/CursesDriver.cs

```diff
@@ -161,6 +160,11 @@ public override void End ()
 			StopReportingMouseMoves ();
 			SetCursorVisibility (CursorVisibility.Default);
 
+			var code = Curses.get_wch (out _);
```

Comment:
Coudn't this be

```cs
while (-1 != Curses.get_wch (out _)) ;
```

?

Reply:
@tig with your suggestion I still received some output:

```bash
dotnet ./UICatalog/bin/Debug/net7.0/UICatalog.dll
^[[<51;71;10m
```

And with the original in the PR I didn't see any output. Try to run UICatalog and open any scenario. Move always the mouse while you press Ctrl-Esc twice to close the app. Was that way I saw the above output.

Reply:
@tig I end up by using the `Curses.flushinp` method to throws away any typeahead that has been typed by the user and has not yet been read by the program.

Done with https://github.com/gui-cs/Terminal.Gui/pull/2786/commits/9ace95bc347674883ab9b5df7d38d6474c27bcfc.